src/Symfony/Bundle/TwigBundle/DependencyInjection/TwigExtension.php

```diff
@@ -78,6 +78,8 @@ public function load(array $configs, ContainerBuilder $container)
             $container->addResource(new DirectoryResource($path));
         }
 
+        $container->setParameter('twig.paths', $config['paths']);
```

Comment:
We do not create parameters for such things. Instead, you need to inject the value directly via `replaceArgument` and add a comment in the XML about the value that is going to be injected (have a look at some other examples in the framework).